mm/src/code/z_bgcheck.c

```diff
@@ -4266,6 +4267,9 @@ u32 SurfaceType_GetEcho(CollisionContext* colCtx, CollisionPoly* poly, s32 bgId)
 }
 
 u32 SurfaceType_IsHookshotSurface(CollisionContext* colCtx, CollisionPoly* poly, s32 bgId) {
+    if (GameInteractor_Should(GI_VB_ENABLE_HOOKSHOT_ANYWHERE, false, NULL))
+        return 1;
+
     return SurfaceType_GetData(colCtx, poly, bgId, 1) >> 17 & 1;
```

Comment:
This way, someone could use the same hook to make _no_ surfaces hookshotable, or maybe _metal_ surfaces hookshotable etc
```suggestion
    return GameInteractor_Should(GI_VB_BE_HOOKSHOT_SURFACE, SurfaceType_GetData(colCtx, poly, bgId, 1) >> 17 & 1, poly, bgId);
```

Reply:
Great idea! Updated with your suggestions.